tests/core/base/_utils/test_get_template_values.py

```diff
@@ -0,0 +1,41 @@
+"""Tests for the get_template_values function in the _utils module."""
```

Comment:
nit: """Tests the `_utils.get_template_values` module."""

Reply:
Done.